RELEASE.md

```diff
@@ -66,24 +96,40 @@ Test published version
 * ensure new project generates
 * ensure old project upgrades nicely
 
-Tag the release
 
-1. Under `Releases` on GitHub choose `Draft New Release`
-* enter the new version number as the tag prefixed with `v` e.g. (`v0.1.12`)
-* for release title choose a great name, no pressure
-* in the description paste the upgrade instructions from the previous release, followed by the new CHANGELOG entry
-* attach the `ember-cli-<version>.tgz` from above
-* click pre-release for beta releases
-* publish the release
-
-[Update ember-new-output and ember-addon-output.](https://gist.github.com/nathanhammond/e0a55b4d0328b45b8ef5)
+## Promote
 
 Announce release!
 
-1. on Twitter
-* then crosslink Twitter post on slack #dev-ember-cli and #ember-cli
+### Create a new release on GitHub
+
+* [Draft a new release.](https://github.com/ember-cli/ember-cli/releases/new)
+  * enter the new version number as the tag prefixed with `v` e.g. (`v0.1.12`)
+  * for release title choose a great name, no pressure
+  * in the description paste the upgrade instructions from the previous release,
+    followed by the new CHANGELOG. entry
+  * attach the `ember-cli-<version>.tgz` from above
+  * Check Pre-release for beta releases.
+  * Publish the release.
+
+### Twitter
+
+> Ember CLI vX.Y.Z "Release name goes here." released!
+https://github.com/ember-cli/ember-cli/releases/tag/vX.Y.Z
+\#emberjs
+
+### Slack
+
+> Ember CLI vX.Y.Z "Release name goes here." released!
```

Comment:
You can just link your tweet and the bot will fill in the text. Then people can easily retweet from slack.